l10n_it_account/models/account_tax.py

```diff
@@ -21,6 +21,20 @@ class AccountTax(models.Model):
         string="Parent Taxes",
     )
 
+    def get_balance_domain(self, state_list, type_list):
+        domain = super(AccountTax, self).get_balance_domain(state_list, type_list)
+        domain.append(
+            (
```

Comment:
Penso sia meglio aggiungere questa tupla solo quando ci sono dei conti da aggiungere al domain, altrimenti si rischia di aggiungere
```
('account_id', 'in', [])
```
che va in AND con il resto del domain e potrebbe escludere delle righe che invece andrebbero incluse.
Quindi in pratica: prima raccogli in una variabile i conti e, se ce ne sono, aggiungi la tupla al domain; che ne pensi?

Reply:
👍🏻